src/Orleans/Streams/QueueAdapters/IQueueAdapterCache.cs

```diff
@@ -34,10 +30,5 @@ public interface IQueueAdapterCache
         /// </summary>
         /// <param name="messages"></param>
         IQueueCache CreateQueueCache(QueueId queueId);
-
-        /// <summary>
-        /// Current total size of this cache.
-        /// </summary>
-        int Size { get; }
```

Comment:
Nothing should be part of the api unless it fit's the usage pattern.  The burden of proof is on the claim that it should be in the api.

What call pattern does Size support?  Users of this interface use it to create caches for specific queues, where does the Size property fit into this call pattern?